### ISSUE TITLE
In GridViewColumnHeader PART_HeaderGripper is not displaying the correct cursor

### ISSUE THREAD

**Issue by Jesus805** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1531]
Hi. 

This problem can be replicated in the MaterialDesign Demo.
In line 91 of _MaterialDesignTheme.ListView.xaml_, there is a thumb `PART_HeaderGripper` that uses the style `MaterialDesignGridColumnHeaderGripper`. In _MaterialDesignTheme.Thumb.xaml_, we can see that `MaterialDesignGridColumnHeaderGripper` is supposed to have cursor SizeWE.
However, if you use Snoop to see the Thumb's cursor you can see that it's set to "None" [image](https://i.imgur.com/aNM87sf.png). This is also evident that the cursor looks different than SizeWE as seen here ![image2](https://i.imgur.com/7Stk6h1.png)
Accidentally named it SideWE in the picture, whoops!

**Comment by Jesus805** [comment]
Been looking into this issue some more and here is what I can conclude.
- The Cursor needs to be set at the ControlTemplate level, setting PART_border to SizeWE will yield the correct cursor when hovering over the thumb. (I had to create a separate ControlTemplate for MaterialDesignGridRowHeaderGripper and MaterialDesignGridColumnHeaderGripper)
- Cursor TemplateBinding has not been working, PART_border will inherit from PART_HeaderGripper which has the cursor set to "None", even if I explicitly set a cursor..
- When dragging the thumb, the cursor will once again go to "None", I'm not sure how to resolve this, I've tried setting this trigger but that didn't help.
```
<ControlTemplate.Triggers>
    <Trigger Property="IsDragging" Value="True">
        <Setter Property="Cursor" Value="SizeWE"/>
    </Trigger>
</ControlTemplate.Triggers>
```
Is it possible to set Mouse.OverrideCursor in a xaml trigger?
